presto-product-tests/src/main/java/io/prestosql/tests/hive/BaseTestHiveViews.java

```diff
@@ -38,7 +38,7 @@
         ImmutableNationTable.class,
         ImmutableOrdersTable.class,
 })
-public class TestHiveViews
+public abstract class BaseTestHiveViews
```

Comment:
I am not super convinced that we need this base class.
Maybe just leave `TestHiveViews` as is, and use it for testing default configuration (with Coral).
And add `TestHiveViewsLegacy` based on how `TestHiveViews` looked before Coral was introduced.

Reply:
That would work too.